src/api/levelpack.js

```diff
@@ -3,6 +3,7 @@ import { forEach } from 'lodash';
 import { authContext } from 'utils/auth';
 import { like, searchLimit, searchOffset } from 'utils/database';
 import { Op } from 'sequelize';
+import * as _ from 'lodash';
```

Comment:
should be imported like: `import { sumBy } from 'lodash';`

Reply:
I figured for server side code this would be ok as it's not sending more data to browser. What do other people think? In some files i end up using a lot of lodash functions. I'm pretty sure its very insignificant overhead on server side code but idk.

Reply:
changes anyways but i don't really see the advantage but that's cool np.